README.md

```diff
@@ -56,80 +63,71 @@ There are four kinds of Gauss-Chebyshev quadrature rules, corresponding to four
-
-* For `n>60`: Use asymptotic expansions for the Legendre nodes and weights <a href="http://epubs.siam.org/doi/abs/10.1137/140954969">[1]</a>.  
+* For `n>60`: Use asymptotic expansions for the Legendre nodes and weights [[1]](http://epubs.siam.org/doi/abs/10.1137/140954969).
 
 ## The algorithm for Gauss-Jacobi
 Gauss quadrature for the weight functions `w(x) = (1-x)^a(1+x)^b`, `a,b>-1`.
 
 *  For `n<=100`: Use Newton's method to solve `Pn(x)=0`. Evaluate `Pn` and `Pn'` by three-term recurrence.
-
-*  For `n>100`: Use Newton's method to solve `Pn(x)=0`. Evaluate `Pn` and `Pn'` by an asymptotic expansion (in the interior of `[-1,1]`) and the three-term recurrence `O(n^-2)` close to the endpoints. (This is a small modification to the algorithm described in <a href="http://epubs.siam.org/doi/abs/10.1137/120889873">[3]</a>.)
-
+*  For `n>100`: Use Newton's method to solve `Pn(x)=0`. Evaluate `Pn` and `Pn'` by an asymptotic expansion (in the interior of `[-1,1]`) and the three-term recurrence `O(n^-2)` close to the endpoints. (This is a small modification to the algorithm described in [[3]](http://epubs.siam.org/doi/abs/10.1137/120889873).)
 * For `max(a,b)>5`: Use the Golub-Welsch algorithm requiring `O(n^2)` operations. 
 
 ## The algorithm for Gauss-Radau
 Gauss quadrature for the weight function `w(x)=1`, except the endpoint `-1` is included as a quadrature node.
 
-The Gauss-Radau nodes and weights can be computed via the `(0,1)` Gauss-Jacobi nodes and weights <a href="http://epubs.siam.org/doi/abs/10.1137/120889873">[3]</a>.
+The Gauss-Radau nodes and weights can be computed via the `(0,1)` Gauss-Jacobi nodes and weights[[3]]("http://epubs.siam.org/doi/abs/10.1137/12088987)>.
 
 ## The algorithm for Gauss-Lobatto
 Gauss quadrature for the weight function `w(x)=1`, except the endpoints `-1` and `1` are included as nodes.
 
-The Gauss-Lobatto nodes and weights can be computed via the `(1,1)` Gauss-Jacobi nodes and weights <a href="http://epubs.siam.org/doi/abs/10.1137/120889873">[3]</a>.
+The Gauss-Lobatto nodes and weights can be computed via the `(1,1)` Gauss-Jacobi nodes and weights[[3]]("http://epubs.siam.org/doi/abs/10.1137/12088987)>.
 
 ## The algorithm for Gauss-Laguerre
 Gauss quadrature for the weight function `w(x) = exp(-x)` on `[0,Inf)`
 
 * For `n<128`: Use the Golub-Welsch algorithm. 
-
-* For `method=GLR`: Use the Glaser-Lui-Rohklin algorithm. Evaluate `Ln` and `Ln'` by using Taylor series expansions near roots generated by solving the second-order differential equation that `Ln` satisfies, see <a href="http://epubs.siam.org/doi/pdf/10.1137/06067016X">[2]</a>.
-
+* For `method=GLR`: Use the Glaser-Lui-Rohklin algorithm. Evaluate `Ln` and `Ln'` by using Taylor series expansions near roots generated by solving the second-order differential equation that `Ln` satisfies, see [[2]](http://epubs.siam.org/doi/pdf/10.1137/06067016X).
```

Comment:
I would say:

Evaluate the Laguerre polynomials `Lₙ` and their derivative by using...

Reply:
It'll be easy to understand! I'll fix it.